package.json

```diff
@@ -19,7 +19,8 @@
     "now-dev": "run -p now-dev:*",
     "now-dev:eleventy": "eleventy --serve --watch --port $PORT",
     "now-dev:js": "run -s dev:js",
-    "now-dev:css": "run -s dev:css"
+    "now-dev:css": "run -s dev:css",
+    "storybook": "start-storybook"
```

Comment:
Maybe we can give this script another name like `document` or something?

Reply:
Also, this is going to conflict with my own PR so I suggest we merge that first and then resolve the conflict in this one since this one has the least changes in this regard.